examples/playbooks/nomatchestest.transformed.yml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
[This](https://github.com/cognifloyd/ansible-lint/blob/yaml-fmt/examples/playbooks/nomatchestest.transformed.yml) is a reformatted copy of [examples/playbooks/nomatchestest.yml](https://github.com/cognifloyd/ansible-lint/blob/yaml-fmt/examples/playbooks/nomatchestest.yml)